src/math/vec.zig

```diff
@@ -70,6 +71,20 @@ pub fn Vec(comptime n_value: usize, comptime Scalar: type) type {
                         .mul(&b.swizzle(.y, .z, .x));
                     return s1.sub(&s2);
                 }
+
+                /// Vector * Matrix multiplication
+                pub inline fn mulMat(vector: *const VecN, matrix: *const mat.Mat(3, 3, Vec(4, T) )) VecN{
```

Comment:
```suggestion
                pub inline fn mulMat(vector: *const VecN, matrix: *math.Mat3x3) VecN{
```

Could we write the type name in these instead so it's easier for users to make sense of it?